### PR TITLE
Separated loading entity names from loading metadata.

### PR DESCRIPTION
**Pull request by harryremon**
- Fixed a bug where the plugin doesn't load in large organizations by separating loading entity names only (on load) and loading entity metadata (on combo-box selection change) (this also optimizes loading speed).

### CODE REVIEW

**Comment by DigitalFlow** [comment]
Hey there, 

thanks for your contribution!
Will have a look at it and merge your changes, if everything is fine :smile:

Kind Regards,
Florian

File: src/lib/MsDyn.Contrib.CloneFieldDefinitions/CloneFieldDefinitionsPlugin.cs [hunk 1]
```diff
@@ -15,7 +15,7 @@ namespace MsDyn.Contrib.CloneFieldDefinitions
...
...RLFrw4hoM+uGTTKR+9KVq4QBFJl42hLrzk0ktnHyn4OWKAbq+lS5d+n5n/LraYCoZTLABlJE8i/zgBzCRJ8Ogjj+K2/3NbppVmyqfz0aHilXXm/c39WS1PIEMe2oMg0wJkM/CckMkgCGPGjEFbWxvGjB6NpubmaEWV9yYP7Y4F4HS0rRayJDAYVNGv2nrqI3PPVblPggzF+brafcs9LB+6tQJ2bN+O9vZ2rF27Fp2dnVGtzUbAwkHUzg4AT6iRZcipGPQUzY6zrMtzM+ZWyx4uSPTDKhwJYSCyB8iS4AezZ8++7kjCzREHdABYsnTpjQnzNzhn/phP230vnAqcZkLQP/nkk/ja176GgXLZ+c0RkxmKEbW/fD8qrtXlYUFEkfhGHDTkn9vc3IKxY0dj5MhRGN7aivr6OkDL1dJ26YUk+LQDO4KU36PdLRDW0jTfp9nBgaIB7Xz2D1zLK/GXvr17sGnjRrS3t2Pdunbs3LkrSMnjqJ0HbwY0x6iT/f/vAcqyFcbRFBpR8Ho1ozSFxFsgpCmQtbqDJLv/i+973/tuPdIwc0QCPYvsSz7AnMxj5uF+sEVo1GOxSzC/nt2O9e7MjNWrV+OmL30JXd3dr6G1lkXoQF1HBaDmiMQjm+abyJ/47zETmlta0NraiuaWFjQ3N6Np2DAkzCaIxlScDhdiaIUQ14b7rgHibG+4bIFpQaqp3CfFGiC+8PsXsHLl03jppU3Y/vJ2UMJunRWxyv6/uKgsiTUDaAHMTG/OIgKbCB2Bm+IDwra+WA6lcFSTRxNnBeAPzByNKYUi2FS/j0h9/CMf+ejPj0S88JEK9B//+Cd/vOqqqx9VwEwonBiUh9GKJq11ATcN0+O2dWh2/4QJ4/G2WbPw618/g57uHh+9lPUgyygzbdbCqBrzXhp2pNXmuPJOn/fK7rR9fasNrwxWsK9/L3p6erB7105s3/EKurq6sG/fPpTLZVQqmSMqsQJx4nT3qmA3nZLLMCwPoFTBRlFpepHt8+Y6RkN9A5qamtDS0oKRo0Zh7NixGDV6FM4880y0DG/BqlWr0NPdE46oauFGo0MnWSuG8+aK0W3pNCuWI9jXtdtU3BYVmAUJwWv6xQppvFBBrCj2+8/8c9x20+x9V2mt//bKK6966kjFyxEb0eW1bNmy7yfGmsoZSHLi5KrSh87V5QWpvPzT1dWFm266Ca
```
Comment (DigitalFlow) [inline src/lib/MsDyn.Contrib.CloneFieldDefinitions/CloneFieldDefinitionsPlugin.cs]: Please leave the name as is, the version number will update during the next release, so we don't need this.

File: src/lib/MsDyn.Contrib.CloneFieldDefinitions/Properties/AssemblyInfo.cs [hunk 2]
```diff
@@ -8,7 +8,7 @@
 [assembly: AssemblyTitle("MsDyn.Contrib.CloneFieldDefinitions")]
 [assembly: AssemblyDescription("")]
 [assembly: AssemblyConfiguration("")]
-[assembly: AssemblyCompany("Florian Kroenert")]
+[assembly: AssemblyCompany("Florian Kroenert (Edited by Remon Kamel)")]
```
Comment (DigitalFlow) [inline src/lib/MsDyn.Contrib.CloneFieldDefinitions/Properties/AssemblyInfo.cs]: While I appreciate your work and would like to mention you as contributor, that won't be inside the AssemblyInfo.
If you're fine with that, I'd add a contributors section to the ReadMe.

**Comment by harryremon** [comment]
Done.
I also fixed a bug where global option-sets failed to create

**Comment by DigitalFlow** [comment]
Great, saw that, thanks a lot!

I just tested it and I am experiencing uncaught exceptions when cloning fields and afterwards changing the source entity. Can you verify that?

Kind Regards,
Florian

**Comment by harryremon** [comment]
Yes,
I've Fixed it, along with another indirect scenario bug.

**Comment by DigitalFlow** [comment]
Great work, thanks a lot!
I will merge and release your changes.

**Comment by DigitalFlow** [comment]
I added you to the contributors section on the ReadMe, hope you're fine with the formatting.

**Comment by harryremon** [comment]
Great, I would appreciate if the "@" mention link to my github profile "https://github.com/harryremon". Other than that, it looks fine. thanks for mentioning me :)

**Comment by DigitalFlow** [comment]
@harryremon: Updated :smile:

**Comment by harryremon** [comment]
Thank you so much, I appreciate it. 😄

**Pull request merged**